lemminx-maven/src/main/java/org/eclipse/lemminx/extensions/maven/searcher/RemoteCentralRepositorySearcher.java

```diff
@@ -52,11 +52,16 @@
 import com.google.gson.JsonObject;
 import com.google.gson.JsonParser;
 
-import io.takari.aether.client.AetherClientAuthentication;
-import io.takari.aether.client.AetherClientConfig;
-import io.takari.aether.client.AetherClientProxy;
-import io.takari.aether.client.Response;
-import io.takari.aether.okhttp.OkHttpAetherClient;
+import org.apache.http.HttpHost;
```

Comment:
What about using the standardest java.net.HttpClient ?

Reply:
Done